_includes/cask.html

```diff
@@ -1,10 +1,7 @@
 
         <td>
-{%- if site.data.cask[include.token] != nil -%}
-            <a href="{{ site.baseurl }}/cask/{{ include.token | uri_escape }}">{{ include.token | escape }}</a>
-{%- else -%}
-            {{ include.token | escape }}
-{%- endif -%}
```

Comment:
Retain this if statement; the non-URL version is used for lists of conflicting casks that may not be in `homebrew/cask`, e.g. https://formulae.brew.sh/cask/1password-cli.
Also, retain the use of `include.token` for the if statement, but use `include.cask.token` for output. This is because the key in the list of `sorted_casks` isn't actually the token (that should probably be renamed), it's a slightly modified version with special characters changed or removed.